### PR TITLE
Update README.md - Add Distributed Execution Platforms.

### PR DESCRIPTION
**Pull request by MichalCz**
Hi there!

I'd like to add Scramjet Platform to the list. The reasoning is that it fits at least two of the three criteria you mentioned (full disclosure: I'm the author, so I can't claim regardless of my personal opinion). The software has won three awards since full launch, at: [CloudX](https://www.developerweek.com/cloudx/awards/#winners), [API World](https://apiworld.co/awards/) and [DeveloperWeek](https://www.developerweek.com/awards/) conferences.

I'd also like to propose to add the "Distributed Execution Platform" category, as our platform is combining some features of stream processing (the software model is based on delivery of streamed data between programs in remote environments), remote execution and workflow platforms and can't be reasonably pushed to a single one of the current categories. I believe Temporal would actually fit quite closely with Scramjet, as through a different set of features they also offer similar value.

I do see the gentle reminder, but we're still small and additional visibility would let us continue bringing a fully featured solution to the open source community, and additional justification to our backers to fund the effort in that direction.

Please let me know of any suggestions and additional information to provide.

### CODE REVIEW

**Comment by v-bus** [comment]
Hi! Why do you call it DEP? It looks like heterogeneous virtualization or heterogeneous cloud computing. And your github project looks old https://github.com/scramjetorg/framework-v4

**Comment by MichalCz** [comment]
@v-bus 

The github project of the platform agent is here and is actively developed: https://github.com/scramjetorg/transform-hub

The framework is stable and is in maintenance mode.

Regarding the naming: the main reason is that we're not aiming this solely for cloud computing, but also for on-premise and localised deployments. The agent is capable of running on something like NanoPi or a Raspberry Pi Zero with just 512 MB of RAM, and the use cases we're after are more on how to glue cloud computing resources with localised systems (imagine something in Retail or Industrial space). The distributed nature of the use case is key here, as we believe that intra-cloud use cases are solved to a certain extent and though could be improved, there's just incremental value we can bring.

To put it in a context - imagine the amount of code to be written when having to do something in the cloud when a physical button is pushed in comparison to turning on a lightbulb in a physical location, when an event in the cloud happens. We're aiming to solve cases similar to the latter.

**Comment by MichalCz** [comment]
And then - looking at it from that perspective you'd think Industrial IoT, or Retail IoT - which again escapes the full picture.

But I'm definitely open to hear any advice as a new category is always an uphill battle. :)

**Comment by v-bus** [comment]
I get it. We developed nanoservices for such solution https://github.com/dmerkushov/nanoservices - better to understand, and newer version https://github.com/dmerkushov/nanoservices2
Main idea to run many nanoservices at any place you can run them. They will be fine while they have network connectivity. We wanted to develop nanoservices skeletons for different languages. On a private production environment, we launched python, node.js and C++ code on many desktops.

**Comment by MichalCz** [comment]
That's really cool - we also have a runner developed for a project that runs C++ code (or any native code packaged as a `.so` or potentially `.dll` - the latter not well tested though) - this is though one of those things that are kinda purpose built. I'll look at nanoservices, as it's possible to build an adapter to it (essentially treat is as docker/k8s replacement in our codebase).

But yeah, essentially we want to run programs in a distributed set of remote environments, hence the "Distributed Execution Platform". You click a button "in the Internet" (or call a public API) and all of the sudden, a couple milliseconds later, a digger in Australia is locked so it can't be stolen. :)

**Comment by MichalCz** [comment]
Hi, just checking in - is it ok to merge?

**Comment by stn1slv** [comment]
Hi @MichalCz ,
I apologize for the late response. I have a few doubts:

1. New categories are welcome, but they should be widely known in the industry (for example, there are analytical reports from Forrester or Gartner for this class of solutions). As for Distributed Execution Platforms applied to integration solutions, it doesn't seem to be the case. Therefore, I suggest adding the solution to an existing category.

2. If it's an OSS-based solution, it's highly recommended to point to the GitHub repository instead of the website link.

**Comment by MichalCz** [comment]
Hi @stn1slv,

Thanks for the remarks - since the Cloud Platform is as of yet not open-source I decided to include the Transform Hub (which is) in the *Workflow Engines* and the platform in *iPaaS* - which would cover the category I wanted to create here.

File: README.md [hunk 1]
```diff
@@ -246,6 +239,7 @@ System integration is the process of linking together different IT systems (com
...
 - [OpenText Alloy](https://businessnetwork.opentext.com/enterprise-data-management/) - A powerful enterprise data management platform that empowers organizations to move beyond basic integration and turn data into insights and action. Software engineers can use this platform to manage data and improve business outcomes.
 - [Oracle Integration Cloud Service](https://www.oracle.com/integration/application-integration/) - A robust platform that accelerates time to go live with pre-built connectivity to any SaaS or on-premises application. Software engineers can use this platform to simplify integration processes and streamline operations.
+- [Scramjet Platform](https://scramjet.org/cloud-platform/) - A lightweight environment enabling seamless data integration and real-time processing across distributed systems. It simplifies complex workflows and stream processing, with standards-based coding patterns and under 5 minute end-to-end setup. Comes in on-premise and SaaS renditions.
```
Comment (stn1slv) [inline README.md:242]: It appears that the specific product does not possess the core capabilities of an **Integration Platform as a Service**. These capabilities include strong support for technical protocol connectors, application adapters for both SaaS and on-premises packaged applications, support for various data formats (such as XML, JSON, ASN.1, etc.), and data standards (such as CIM, EDIFACT, HL7, SWIFT, etc). Additionally, an Integration Platform as a Service should offer orchestration, choreography, data validation, and transformation functionalities.

Comment (stn1slv) [reply README.md:242]: Perhaps, the product's main focus is not on system integration.

File: README.md [hunk 2]
```diff
@@ -365,6 +359,8 @@ System integration is the process of linking together different IT systems (com
 - [Elsa Core (⭐5.5k)](https://github.com/elsa-workflows/elsa-core) - A library that enables workflow execution in any .NET Core application, making it easier to automate business processes in .NET environments.
 - [Flowable (⭐7.2k)](https://github.com/flowable/flowable-engine) - A set of open-source business process engines that are compact and efficient, making it easier to automate business processes in a scalable and reliable way.
 - [jBPM (⭐1.5k)](https://github.com/kiegroup/jbpm) - A toolkit for building business applications that helps you automate business processes and decisions, making it easier to manage complex workflows.
+- [Scramjet Transform Hub(⭐64)](https://github.com/scramjetorg/transform-hub) - A lightweight FaaS execution engine for running
+long lived functions in Python and JavaScript with Docker, Kubernetes and Process backends.
```
[the remark this answers is not in the document]
Comment (stn1slv) [reply README.md:363]: Which workflow management capabilities does it provide? For example, long-running and stateful, timers, visual workflow management user interfaces, support workflow management notations (like BPMN, CMMN, DMN) or custom ones, etc.

**Review by stn1slv — CHANGES_REQUESTED** [review]

**Pull request closed without merging**